devel/build

```diff
@@ -25,7 +25,7 @@ export GIT_REVISION=$(git describe --tags --abbrev=40 --always --dirty || true)
 # Calling `docker run nextstrain/base` will still only pull down the small base
 # image rather than pulling down the larger nextstrain/base-builder image.
 
-platform=linux/amd64
+platform=linux/amd64,linux/arm64
```

Comment:
This only works when emulation is configured properly (done for GitHub Actions via [`docker/setup-qemu-action`](https://github.com/docker/setup-qemu-action)). This means it won't be as portable to individual dev machines. Two options here:

1. Add emulation setup as a prerequisite to the build instructions in the README.
2. Make `devel/build` accept platforms as a parameter, and only use emulation when necessary.

I'm leaning towards (2) because I image there would be cases where one would want to build for just a single platform locally since it doesn't require emulation setup, and it will build much faster than a multi-platform build.

Reply:
(2) sounds good to me.

Reply:
Done: 7efb3432b7b9aa5e8d12db229550058e985bebb5

Added an entry in testing checklist for manually checking that single-platform build still works locally.